deploy/helm/quarks/README.md

```diff
@@ -21,14 +21,15 @@ The operator will watch for BOSH deployments in separate namespaces (default: on
 
 ### Using multiple operators
 
-Choose different namespaces and cluster role names. The persist output service account will be named the same as the cluster role:
+Choose different namespaces and cluster role names. The persist output service account will be named the same as the cluster role as well as for coredns:
 
 ```
 helm install qops1 quarks/quarks \
   --namespace namespace1
   --set "global.singleNamespace.name=staging1" \
   --set "global.monitoredID=id1" \
   --set "quarks-job.persistOutputClusterRole.name=clusterrole1" \
+  --et  "corednsServiceAccount.name=clusterrole2" \
 ```
 
 ### Using multiple namespaces with one operator
```

Comment:
I guess we need also to update the docs in the website :+1:

Reply:
Done. https://github.com/cloudfoundry-incubator/quarks-docs/pull/28